src/jsifier.mjs

```diff
@@ -45,6 +45,10 @@ const addedLibraryItems = {};
 
 const extraLibraryFuncs = [];
 
+// Experimental feature to check for invalid __deps entries. Modify this line
+// locally to try it out.
```

Comment:
Is the comment stale? Now it seems there is an env var, no need to modify the source line?